blog/content/post/2024-07-23-new-mtl-releases.md

```diff
@@ -0,0 +1,310 @@
+profiles.
+![Fan profiles](/img/fan_profile.png)
+
+### [Fn lock hotkey feature](https://docs.dasharo.com/unified/novacustom/features/#fn-lock-hotkey)
+
+By default using the F1-F12 keys for additional actions like changing
+the screen brightness or volume requires holding the Fn key.
+Now you can Lock the Fn key so that you things like muting your
+microphone can be done without holding down the Fn key.
+
+### [Throttling temperature adjustment in setup menu](https://docs.dasharo.com/unified/novacustom/features/#cpu-throttling-threshold)
+
+By setting the CPU throttling threshould you can specify what
+CPU temperatures are acceptable for your device. When the
+cooling system does not manage to keep the CPU below this
+temperature the CPU will be throttled to prevent overheating.
+![Throttling temperature adjustment](/img/cpu_throttling_threshold.png)
+
+## Known issues
+
+Sadly there are some issues that were not resolved in time for the release.
+Here is a list of the issues on which we are working on:
+
+- [No HDMI output in FW on V540TU and V560TU](https://github.com/Dasharo/dasharo-issues/issues/930)
+- [Laggy behaviour on Manjaro (KDE) with open drivers](https://github.com/Dasharo/dasharo-issues/issues/911)
+- [V540TU: Option Previous power state restoration doesn't work](https://github.com/Dasharo/dasharo-issues/issues/931)
+- [Artifacts in video playback in some players using HW acceleration](https://github.com/Dasharo/dasharo-issues/issues/948)
+- [Only native resolution listed for internal panel](https://github.com/Dasharo/dasharo-issues/issues/949)
+
+## Test coverage
+
+The tests were performed on NovaCustom V540TU and V560TU platforms.
+Checkout the results at our
+[osfv-results repo](https://github.com/Dasharo/osfv-results):
+
+- [V540TU](https://github.com/Dasharo/osfv-results/blob/main/boards/NovaCustom/MTL_14th_Gen/V540TU/v0.9.0-results.csv)
+- [V560TU](https://github.com/Dasharo/osfv-results/blob/main/boards/NovaCustom/MTL_14th_Gen/V560TU/v0.9.0-results.csv)
+
+The validation procedure is described at [laboratory assembly guide](https://docs.dasharo.com/transparent-validation/novacustom/laboratory-assembly-guide/#prerequisites).
+Details about specific test cases can be found in our [Dasharo Test Specification]\
+(<https://docs.dasharo.com/unified-test-documentation/overview/>)
+
+## Our contributions
+
+While working on the new release we have made multiple contributions
+open source projects:
+
+### Coreboot
+
+- <https://review.coreboot.org/c/coreboot/+/82671>
+- <https://review.coreboot.org/c/coreboot/+/82672>
+- <https://review.coreboot.org/c/coreboot/+/82673>
+- <https://review.coreboot.org/c/coreboot/+/82686>
+- <https://review.coreboot.org/c/coreboot/+/82685>
+- <https://review.coreboot.org/c/coreboot/+/82674>
+- <https://review.coreboot.org/c/coreboot/+/82898>
+
+### Linux kernel
+
+- <https://github.com/torvalds/linux/commit/e1c6db864599be341cd3bcc041540383215ce05e>
+
```

Comment:
We can add these 2 to the list:
- https://github.com/MrChromebox/edk2/commit/7f398d3b14928ffecbbe92bd93213db91dad7703
- https://github.com/systemd/systemd/commit/f5c8dd85ee13f9308498faf6a0c4837e604f8dcb

Reply:
added